media/css/custom.scss

```diff
@@ -521,3 +526,32 @@ summary {
     color: $white;
   }
 }
+
+.dictionary-item a {
```

Comment:
Why `item`? `container` would be much better. Nest the definition of `a` under `.dictionary-container` as well.